components/popover/mobile-scroll-lock.js

```diff
@@ -0,0 +1,129 @@
+/**
```

Comment:
There is a folder with all Higher-Order Components: https://github.com/WordPress/gutenberg/tree/master/components/higher-order. We also try to come up with a pattern for naming such helpers. It seems like in this case `withScrollLock` would be a good pick. In general, we prefix all HOCs with `with` in the majority of cases. The only exception is when a component might not be rendered when a given condition occurs. In such case `if` is the preferred prefix. @aduth might even update our docs somewhere, but I'm sure he commented about it at least twice.

Reply:
Okey, I now see that it is an independent component. Skip my previous comment in this context :)